### PR TITLE
✨️ Add feature to remove SSL from site command

### PR DESCRIPTION
**Pull request by kirtangajjar**

### CODE REVIEW

File: src/helper/class-ee-site.php [hunk 1]
```diff
@@ -857,16 +881,42 @@ protected function update_ssl( $assoc_args ) {
...
 	}
 
+	/**
+	 * Disables SSL on a site.
+	 *
+	 * @throws \Exception
+	 */
+	private function disable_ssl() {
+
+		$this->dump_docker_compose_yml([ 'nohttps' => 'nohttps' ]);
```
Comment (mrrobot47) [inline src/helper/class-ee-site.php:914]: Please keep the `nohttps` value boolean like in the rest of the file.

**Review by mrrobot47 — CHANGES_REQUESTED** [review]
Have you checked the scenario where you re-enable SSL after switching it off?
Is it using the existing certificate or requesting a new one?
If it is using the existing one, is it checking for expiry or not?

File: src/helper/class-ee-site.php [hunk 2]
```diff
@@ -357,8 +357,16 @@ protected function delete_site( $level, $site_url, $site_fs_path, $db_data = []
...
-	 * [--ssl=<ssl>]
+	 * [--ssl[=<ssl>]]
 	 * : Enable ssl on site
+	 * ---
+	 * options:
+	 *   - le
+	 *   - self
+	 *   - inherit
+	 *   - custom
+	 *   - "off"
```
[the remark this answers is not in the document]
Comment (kirtangajjar) [reply src/helper/class-ee-site.php:368]: This is required, as unquoted "off" is interpreted as false(boolean) by our YAML parser.

[the remark this answers is not in the document]
Comment (kirtangajjar) [reply src/helper/class-ee-site.php:368]: Yes, I have checked the scenario of re-enabling SSL after switching it off. It uses the existing certificate, and also checks for expiry.

**Review by mrrobot47 — APPROVED** [review]

**Pull request merged**